plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterService.java

```diff
@@ -92,6 +92,30 @@ public interface KubernetesClusterService extends PluggableService, Configurable
             true,
             ConfigKey.Scope.Account,
             KubernetesServiceEnabled.key());
+    static final ConfigKey<Long> KubernetesControlNodeInstallAttemptWait = new ConfigKey<Long>("Advanced", Long.class,
```

Comment:
I think these settings shouldn't be part of this PR but for #9102

Reply:
I believe this is just externalizing the already present in the CKS scripts. So it wouldn't cause any issues. Unless you still want to me to revert

Reply:
IMO we should fix only NSX related issues on this PR and we could add the externalize settings on the other PR